.gitignore

```diff
@@ -1,5 +1,9 @@
 coverage.out
 
+.idea
 /bin
 /dist
 /hugo
+
+/node_modules
+/assets
```

Comment:
Instead of committing this, I'd rather configure a global `.gitignore`:

```console
git config --global core.excludesfile ~/.gitignore
```

[This is mine.](https://gist.github.com/refs/8b3280602e24bdc10cbb25c012a628f7)

Reply:
☝️ /cc @kulmann